heroku-16.dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM heroku/heroku:16
+MAINTAINER Akash Manohar
+
+RUN apt-get update
+
+RUN apt-get install -y curl wget ca-certificates
+RUN apt-get install -y gcc g++
+RUN apt-get install -y make automake autoconf
+RUN apt-get install -y libwxgtk3.0-dev libgl1-mesa-dev libglu1-mesa-dev libpng3
```

Comment:
The only real difference between this and the [cedar-14.dockerfile](https://github.com/HashNuke/heroku-buildpack-elixir-otp-builds/blob/master/cedar-14.dockerfile) is that I needed to use `libwxgtk3.0-dev` instead of `libwxgtk2.8-dev` to get the Docker build to complete successfully.